crates/uv/tests/tool_list.rs

```diff
@@ -155,9 +166,12 @@ fn tool_list_bad_environment() -> Result<()> {
     success: true
     exit_code: 0
     ----- stdout -----
+    The following tools are already installed:
+
     ruff v0.3.4
     - ruff
 
+
```

Comment:
Are we adding a trailing newline to the output?

Reply:
This is probably slipped through the cracks. I will remove it.